SF_Users_API/db/migrations/20171030225656_initial.js

```diff
@@ -0,0 +1,33 @@
+
+exports.up = function(knex, Promise) {
+  return Promise.all([
+    knex.schema.createTable('sf_users', function(table) {
+      table.increments('id').primary();
+      table.string('username');
+      table.string('email');
+      table.string('address');
```

Comment:
@danalvarez5280 Can we call this 'street_address'?  It will be more symantic for what the data is actually meant to include.